### ISSUE TITLE
mvdXML 1.2 grammar mistake

### ISSUE THREAD

**Issue by pjanck** [buildingSMART/mvdXML#13]
Currently:

```
boolean_expression
: ( NOT ) boolean_term (logical_interconnection ( NOT ) boolean_term)* ;
```

Missing `?` for the `(NOT)` part - or is it a must?

Proposal:

```
boolean_expression
: ( NOT )? boolean_term (logical_interconnection ( NOT )? boolean_term)* ;
```

**Comment by pjanck** [comment]
I can prepare a PR after #19 and #21 have been merged (for easier traceability).

**Comment by MatthiasWeise** [comment]
Closed by #21
